Add find_many_by_ids to wings query_service

hackmastera opened the pull request:
This is a naive implementation, which valkyrie uses in its fedora
adapter. Another implementation was explored, using https://github.com/samvera/active_fedora/blob/master/lib/active_fedora/relation/finder_methods.rb#L94, but that returns SolrHit objects which we believe we'd need to call ActiveFedora.find over, anyway.

fixes #3614

no-reply commented on lib/wings/valkyrie/query_service.rb:
```diff
@@ -36,6 +36,17 @@ def find_all
+          rescue ::Valkyrie::Persistence::ObjectNotFoundError
+            nil
+          end
+        end.reject(&:nil?)
```
`.compact` might be nicer?

hackmastera commented:
@no-reply I added a new commit that uses `ActiveFedora::Base.where`; because `ActiveFedora::Base.find` with multiple ids throws an error if one of the ids isn't found, which is against the Valkyrie API. If you want to mess around with other implementations, feel free. I added this new option as a new commit so it can easily rolled back if you prefer the naive implementation.
